src/gui/sdlmain.cpp

```diff
@@ -1965,7 +1965,9 @@ Bitu GFX_SetSize(int width,
 		// Log changes to the rendering driver
 		static std::string render_driver = {};
 		if (render_driver != rinfo.name) {
-			LOG_MSG("SDL: Using driver \"%s\" for texture renderer", rinfo.name);
+			LOG_MSG("SDL: Using \"%s\" driver for %d-bit texture rendering",
```

Comment:
Very minor: usually I just use single quotes in the log messages

Reply:
fixed.